filesec/file_security.go

```diff
@@ -69,6 +69,9 @@ type Config struct {
 
 	// DisableTLSVerify disables TLS verify in HTTP clients etc
 	DisableTLSVerify bool
+
+	// Support always overwriting the local filesystem cache
```

Comment:
`// AlwaysOverwriteCache supports always overwriting the local filesystem cache`